groups/data-reconciliation/module-iam/main.tf

```diff
@@ -83,3 +83,51 @@ data "aws_iam_policy_document" "cloudwatch-policy" {
   }
 }
 
+# IAM user responsible for uploading results to S3
+resource "aws_iam_user" "iam-user" {
```

Comment:
Could this be done through the task role instead of defining a user to do so?

Reply:
The application requires an access key id and a secret access key in order to work.

Reply:
I've raised BI-8463 to rework the application to use AWS_CONTAINER_CREDENTIALS_RELATIVE_URI.

Reply:
If we put a story in the backlog to revisit this that would be fine